clang_tidy.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Lets' move this file to the scripts folder.

Reply:
Done